templates/common/infra/bicep/core/host/container-app.bicep

```diff
@@ -22,7 +44,13 @@ resource app 'Microsoft.App/containerApps@2022-03-01' = {
   name: name
   location: location
   tags: tags
-  identity: { type: managedIdentity ? 'SystemAssigned' : 'None' }
+  identity: managedIdentityEnabled || managedIdentity ? {
+    type: 'SystemAssigned,UserAssigned'
```

Comment:
is it ok to always have `UserAssigned` here even if it's only system assigned identity?

Reply:
For this case, I can add a conditional ternary operator again checking on the managedIdentity name, if provided assign both `UserAssigned` else if managed identity alone is enabled use `SystemAssigned` else have it as none. 
There is a behavior that I noticed that if both `SystemAssigned,UserAssigned` is used, `SystemAssigned` take more priority and that might be an issue.